.github/workflows/release-30_publish_release_draft.yml

```
@@ -23,10 +23,31 @@ jobs:
           echo "stable=$RUST_STABLE_VERSION" >> $GITHUB_OUTPUT
 
   build-runtimes:
-    uses: "./.github/workflows/srtool.yml"
+    uses: "./.github/workflows/release-srtool.yml"
     with:
        excluded_runtimes: "substrate-test bp cumulus-test kitchensink minimal-template parachain-template penpal polkadot-test seedling shell frame-try sp solochain-template"
 
+  build-frame-omni-bencher:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+
+      - name: Build frame-omni-bencher binary
+        run: |
+          sudo apt update
+          sudo apt install -y protobuf-compiler
+
+          cargo build --locked --release -p frame-omni-bencher
+          target/release/frame-omni-bencher --version
```

Let's use production profile

```suggestion
          cargo build --locked --profile=production -p frame-omni-bencher
          target/production/frame-omni-bencher --version
```